hathor/cli/nginx_config.py

```diff
@@ -192,10 +194,16 @@ def generate_nginx_config(openapi: Dict[str, Any], *, out_file: TextIO, rate_k:
     #      another possible reasoning considered was estimating the client capacity per node based on the parameters
     #      already set on http endpoints, but the fact that request limiting can have bursts, and also that the rate of
     #      global vs per ip vary a lot, this was nor persuited.
-    websocket_max_conn_global = 4000
-    websocket_max_conn_per_ip = 10
-    mining_websocket_max_conn_global = 100
-    mining_websocket_max_conn_per_ip = 4
+    if disable_rate_limits:
+        websocket_max_conn_global = 10000
+        websocket_max_conn_per_ip = 10000
+        mining_websocket_max_conn_global = 1000
+        mining_websocket_max_conn_per_ip = 1000
+    else:
+        websocket_max_conn_global = 4000
+        websocket_max_conn_per_ip = 10
+        mining_websocket_max_conn_global = 100
+        mining_websocket_max_conn_per_ip = 4
```

Comment:
I know this isn't great, but I think it's fine for now. Maybe we should create an issue to use jinja2 for the nginx config file, and also possibly move this out of this repo.

Reply:
I have the same feeling that we should move it out of this repo. But I also feel that it might be a good recommendation for anyone running a public node to use these limits.

Reply:
Created an issue for using templating when generating the file: https://github.com/HathorNetwork/hathor-core/issues/265